mysql-test/suite/rocksdb/t/i_s_ddl.test

```diff
@@ -17,6 +17,7 @@ CREATE TABLE is_ddl_t2 (x INT, y INT, z INT,
                         PRIMARY KEY (z, y) COMMENT 'zy_cf',
                         KEY (x)) ENGINE = ROCKSDB;
 
+--sorted_result
```

Comment:
A potential ORDER BY issue would be lack of unique key and too many potentially equal columns, resulting in very long ORDER BY, and I don't know the TokuDB issue, which could have been something legitimate too. Otherwise I do prefer SQL over MTR sort